.github/workflows/test.yml

```diff
@@ -30,11 +30,13 @@ jobs:
         run: |
           pip3 install lxml
 
-      - name: Run Tests
+      - name: Test nodl
         uses: ros-tooling/action-ros-ci@0.0.16
         id: action_ros_ci_step
         with:
           package-name: nodl_python ros2nodl
+          colcon-mixin-name: coverage-pytest
+          colcon-mixin-repository: https://raw.githubusercontent.com/colcon/colcon-mixin-repository/5c45b95018788deff62202aaa831ad4c20ebe2c6/index.yaml
```

Comment:
What is this?

Reply:
This adds the invocation to run coverage 

https://github.com/colcon/colcon-mixin-repository/blob/1ddb69bedfd1f04c2f000e95452f7c24a4d6176b/coverage.mixin#L9-L18

Reply:
Ah ha, okay.